recipe/meta.yaml

```diff
@@ -31,16 +39,15 @@ test:
     - pyasn1.type
 
 about:
-  home: https://github.com/etingof/pyasn1
-  license: BSD 2-Clause
+  home: https://github.com/pyasn1/pyasn1
```

Comment:
FYI, the explanation of why the maintainer has been changed https://github.com/pypi/support/issues/2090. 
Changing the repo maintainer can be qualified as a security concern and should be always investigated by us.

Reply:
Thank you! I didn't know, and I would have known where to search. I'll make sure to do that in the future.